### ISSUE TITLE
Maps exported as .vxl do not work in Ace of Spades servers

### ISSUE THREAD

**Issue by Bossieh** [xtreme8000/goxel#3]
Hello,

First of all thanks a lot for adding the export to .vxl feature!

But when I start a server with the .vxl file exported from Goxel the (OpenSpades) clients cannot connect. The OpenSpades client hangs on "Still loading..."

Any idea?

**Comment by xtreme8000** [comment]
The export function was made to work with arbitrary sizes. Before export, make sure your map is 512x512 and at most 64 blocks tall. Meaning there must be at least a solid block floor in the 512x512 plane.

**Comment by Bossieh** [comment]
Thanks for your quick answer.

I used a fork made by Bytebit (goxel_aos_6bc8b), and seems like that version can export useable .vxl files.

https://gamebanana.com/tools/8108

**Comment by xtreme8000** [comment]
yes that's also me, I did not upload it to that site however. The zip on there is a cross-build of this goxel-fork that I published to discord last year.
